theme/valo/vaadin-combo-box.html

```diff
@@ -1,63 +1,26 @@
-<link rel="import" href="../../../vaadin-valo-theme/color.html">
-<link rel="import" href="../../../vaadin-valo-theme/spacing.html">
 <link rel="import" href="../../../vaadin-valo-theme/font-icons.html">
+<link rel="import" href="../../../vaadin-valo-theme/mixins/field-button.html">
 
 <link rel="import" href="../../../vaadin-text-field/theme/valo/vaadin-text-field.html">
 <link rel="import" href="vaadin-combo-box-dropdown.html">
 <link rel="import" href="vaadin-combo-box-item.html">
 
 <dom-module id="valo-combo-box" theme-for="vaadin-combo-box">
   <template>
-    <style>
-      :host([label]) {
-        margin-top: var(--valo-space-m);
-        vertical-align: var(---vaadin-text-field-vertical-align-offset);
-      }
-
-      [part="text-field"] {
-        display: block;
-        max-width: 100%;
-      }
-
-      [part$="button"] {
-        width: 24px;
-        height: 24px;
-        line-height: 24px;
-        font-size: 24px;
-        text-align: center;
-        color: var(--valo-contrast-40pct);
-        transition: 0.2s color;
-        cursor: default;
-      }
-
-      [part$="button"]:hover {
-        color: var(--valo-contrast-70pct);
-      }
-
-      [part$="button"]::before {
-        font-family: "valo-icons";
-      }
-
-      [part="toggle-button"]::before {
-        content: "\e905";
-      }
-
-      [part="clear-button"]::before {
-        content: "\e907";
-      }
-
-      /* Read-only */
-
-      :host([readonly]) [part="toggle-button"] {
-        display: none;
-      }
-
-      /* Disabled */
-
-      :host([disabled]) [part$="button"] {
-        color: var(--valo-contrast-20pct);
-      }
-    </style>
+  <style include="valo-field-button">
+    [part="text-field"] {
+      /* TODO Relies on knowing the width and padding sizes of vaadin-text-field */
+      width: calc(10em + (0.25em + 0.375em + var(--valo-border-radius) / 4) * 2);
```

Comment:
I actually found a nicer way to size the text-field, so this won’t be necessary. Will send a PR for text field soon.